.env.example

```
@@ -12,9 +12,9 @@ SERVER_BASE_PATH=/api
 SERVER_READ_TIMEOUT=5s
 SERVER_ALLOWED_ORIGINS=
 SERVER_MAX_REQUEST_BODY_BYTES=20971520
-SERVER_TLS_CERTFILE=
-SERVER_TLS_KEYFILE=
-SERVER_ENABLE_TLS=false
+SERVER_TLS_CERTFILE=cert.pem
+SERVER_TLS_KEYFILE=key.pem
+SERVER_ENABLE_TLS=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVER_ENABLE_TLS key should go before the SERVER_HOST key

(UnorderedKey)